src/cc_mm.c

```diff
@@ -82,6 +82,22 @@ _cc_realloc(void *ptr, size_t size, const char *name, int line)
     return p;
 }
 
+void *
+_cc_realloc_test(void *ptr, size_t size, const char *name, int line)
```

Comment:
Not a huge fan of the function name, which does not describe the *nature* of the difference, how about `_cc_realloc_move`?